config.yaml

```diff
@@ -22,6 +20,9 @@ default:
  commit_updates: True
  push_updates: True
  force_rebuild: False
+release_notes:
+ release_notes_repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
+ release_notes_repo_branch: "master"
```

Comment:
Do we need to repeat the name of the root node in its elements?  For example, `relete_notes_repo_url` is inside `release_notes`, it could be just `repo_url`.  Something like this:

```yaml
release_notes:
  repo_url:
  repo_branch:
```